tensorizer/serialization.py

```diff
@@ -190,7 +159,7 @@ def __init__(
         ],
         device: Union[torch.device, str, None] = None,
         filter_func: Optional[Callable[[str], Union[bool, Any]]] = None,
-        dtype: Union[numpy.dtype, str, None] = None,
+        dtype: torch.dtype = None,
```

Comment:
It's important to annotate that this parameter is optional.
```suggestion
        dtype: Optional[torch.dtype] = None,
```